changelog/QEz6SICbSKaCXgVTXqYGjg.md

```diff
@@ -0,0 +1,4 @@
+audience: users
+level: minor
+---
+Improve cache mounting speed on linux, especially when they contain a lot of tiny files
```

Comment:
```suggestion
Generic Worker: Improve cache mounting speed on linux, especially when they contain a lot of tiny files
```

Reply:
Added

Reply:
Awesome thanks. I'll set up for automerge now 👍🏻